docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -86,27 +86,20 @@ It also enables you to **stack** them on top of one another (although this is di
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
 
-- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert` through the SnackbarContent `role` prop. This is the default role.
-- If a Snackbar requires focus to close it, then content authors should use the `role` of `alertdialog`.
+- By default, the toast won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended giving users [enough time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html).
+- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert="role"` through the SnackbarContent. This is the default role.
```

Comment:
```suggestion
- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `role="alert"` through the SnackbarContent. This is the default role.
```

Shouldn't the default role and default auto-hide behaviour match?

Reply:
I believe the snackbar doesn't require the user to close it, by default. I think that it's meant in a context where the dialog blocks interactions with the rest of the page.

Reply:
It could probably do with being reworded in that case, asa it's ambiguous at the moment, particularly in the context of the previous sentence about autohide.